src/figuresClasses.ts

```diff
@@ -1,19 +1,72 @@
 export interface Figure {
-
-}
+  shape: string,
```

Comment:
It would be good to create a type for shape and color, for example 

`type Colors = 'red' | 'green' | ... `